scratchapi2/misc.py

```diff
@@ -101,3 +101,14 @@ def offline_ver():
                           raw_xml)
         val = match.group(1)
         return val
+
+    @staticmethod
+    def save_asset(asset_name, filename_or_obj):
+        """Save asset to a file. asset_name must be with an extension."""
+        if isinstance(filename_or_obj, str):
+            filename_or_obj = open(filename_or_obj, 'wb')
+        _streaming_request(filename_or_obj,

```

Comment:
Please remember to close file objects. Convert this:
```python
_streaming_request(...)
```
to this:
```python
with filename_or_obj:
    _streaming_request(...)
```

Reply:
Oops!